README.md

```diff
@@ -23,3 +52,4 @@ A list of resources for data science work on climate change
 ## Meta
 
 * [modernising energy data wiki](https://modernisingenergydata.atlassian.net/wiki/spaces/MED/overview)
+* [Building Performance Network project map](https://building-performance.network/news/housing-performance-map-add-a-project)
```

Comment:
can you change this link from teh survey to the actual public facing data set please? https://building-performance.network/resource/housing-performance-map